utils/build-apple-framework.sh

```diff
@@ -27,6 +27,13 @@ fi
 
 # Utility function to configure an Apple framework
 function configure_apple_framework {
+  local build_cli_tools
+  if [[ $1 == iphone* ]]; then
+    build_cli_tools="false"
+  else
+    build_cli_tools="true"
+  fi
```

Comment:
Personally I think having the `if` check for `mac` and explicitly setting it to `true` (`false` otherwise) would be a bit more future proof when we add e.g. Apple Watch.

Reply:
Good point 👍